org.envirocar.core/src/main/java/org/envirocar/core/utils/ServiceUtils.java

```diff
@@ -52,15 +52,21 @@ public static final void startService(Context context, Intent serviceIntent) {
     }
 
     public static final void stopService(Context context, Class<?> service) {
-        LOGGER.info("Trying to stop %s", service.getSimpleName());
+        Intent serviceIntentstop = new Intent(context, service);
+        startService(context, serviceIntentstop);
```

Comment:
Seems like a copy&paste mistake. Please, be a little more attentive when changing the code in order to reduce the number of review cycles.

Reply:
Updated